src/_nebari/upgrade.py

```diff
@@ -230,7 +230,15 @@ def requires_nebari_version_field(self):
         """
         return rounded_ver_parse(self.version) > rounded_ver_parse("0.3.13")
 
-    def upgrade_step(self, config, start_version, config_filename, *args, **kwargs):
+    def upgrade_step(
+        self,
+        config,
+        start_version,
+        config_filename,
+        *args,
+        attempt_fixes=False,
+        **kwargs,
```

Comment:
We should already have access to `attempt_fixes` in `kwargs`. See:

https://github.com/nebari-dev/nebari/blob/3f2523aae801fa5d09c414aec6fe1eb237effc0f/src/_nebari/upgrade.py#L485-L494

Did you add this because that does not work as expected?

Reply:
> Did you add this because that does not work as expected?

No, I should have used it from `kwargs`. Done.